### PR TITLE
Update whats-new-in-defender-for-office-365.md

### PR DESCRIPTION
**Pull request by MishraSoumyaMS**

### CODE REVIEW

File: microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md [hunk 1]
```diff
@@ -45,7 +45,10 @@ For more information on what's new with other Microsoft Defender security produc
 - [What's new in Microsoft Defender for Endpoint](../defender-endpoint/whats-new-in-microsoft-defender-endpoint.md)
 - [What's new in Microsoft Defender for Identity](/defender-for-identity/whats-new)
 - [What's new in Microsoft Cloud App Security](/cloud-app-security/release-notes)
+## April 2022
 
+- [Manual email remediation enhancements](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/remediate-malicious-email-delivered-office-365?view=o365-worldwide): Bringing manual e-mail purge actions taken in Microsoft Defender for Office 365 to the Microsoft 365 Defender (M365D) unified Action Center using a new action focused investigation.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md]:
```suggestion
- [Manual email remediation enhancements](/microsoft-365/security/office-365-security/remediate-malicious-email-delivered-office-365): Bringing manual email purge actions taken in Microsoft Defender for Office 365 to the Microsoft 365 Defender (M365D) unified Action Center using a new action-focused investigation.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MishraSoumyaMS, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Review by MSFTTracyP — APPROVED** [review]
Thanks!

File: microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md (hunk 1 above)
Comment (MSFTTracyP) [reply microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md]: @JohanFreelancer9, this was supposed to be published days ago. The updates are rolling live. Please let's get done with edits and push live.

**Comment by MSFTTracyP** [comment]
Please push this documentation change live immediately, as this change is rolling out and its blog is public.

**Pull request merged**